app/konnect/configure/runtime-manager/index.md

```diff
@@ -3,19 +3,27 @@ title: Runtime Manager Overview
 no_version: true
 ---
 
-The Runtime Manager is a {{site.konnect_saas}} functionality module
-that lets you catalogue, connect to, and monitor the status of all runtimes in
-one place.
+The [Runtime Manager](https://konnect.konghq.com/runtime-manager)
+is a {{site.konnect_saas}} functionality module
+that lets you catalogue, connect to, and monitor the status of all runtime
+groups and instances in one place, as well as manage group configuration.
 
-With {{site.konnect_short_name}} acting as the control plane, a runtime
+The Runtime Manager overview page displays a list of
+runtime groups currently owned by the organization. From here, you can add or
+delete runtime groups, or go into each individual group to manage runtime
+instances and their global configuration.
+
+_[screenshot of overview page]_
+
+With {{site.konnect_short_name}} acting as the control plane, a runtime instance
 doesn't need a database to store configuration data. Instead, configuration
-is stored in-memory on each node, and you can easily update multiple runtimes
-from one {{site.konnect_short_name}} account with a few clicks.
+is stored in-memory on each node, and you can easily update multiple runtime
+instances from one {{site.konnect_short_name}} account with a few clicks.
```

Comment:
To avoid overloading the terms `multiple runtimes` and `multiple runtime groups`, I've heard Reza use "runtime clusters" to describe the same thing. 

Just a tiny suggestion, use if you like it.
```suggestion
is stored in-memory on each node, and you can easily update clusters of runtimes from one {{site.konnect_short_name}} account with a few clicks.
```

Reply:
Isn't the collective name for multiple runtime instances a runtime group? (technically a RG is the CP, but in practice it's all config for a set of instances?

Reply:
This sentence made more sense back when it was describing a pre-RG environment. I do want to get across the idea that you can apply config to lots of instances at once, so maybe something like this:

> you can easily update all runtime instances in a group with a few clicks.